OpenXMLSDK.Engine/Word/ReportEngine/Models/Label.cs

```diff
@@ -26,6 +26,11 @@ public class Label : BaseElement
         /// </summary>
         public bool IsPageNumber { get; set; }
 
+        /// <summary>
+        /// if true, the label make a tabulation
+        /// </summary>
+        public bool IsTabulation { get; set; }
```

Comment:
If you create a Tabulation object, why do you add this field ?

Reply:
The render of the TabulationObject can be a new Run("     ") instead update the sub Label field and render it.

Reply:
Tabulation is now a property of a label instead of a new object. A TabChar is added to the label run instead of the text. If a label contains tabulation properties, the text is not displayed.